### ISSUE TITLE
Email piping to a script

### ISSUE THREAD

**Issue by s-a-s-k-i-a** [EasyEngine/easyengine#958]
Currently easy engine offers no easy solution to integrate email piping to script (is needed to integrate support ticket tools and CRMs for example).

VimbAdmin developers consider this feature out of scope for VimbAdmin. ( rel: https://github.com/opensolutions/ViMbAdmin/issues/155 )

So I think there should be some kind of other simple solution, every user can use without fiddling with postfix config files.

Thanks

**Comment by rahul286** [comment]
v4 wont support webmail. Sorry.

**Comment by s-a-s-k-i-a** [comment]
This is not a webmail related issue.

**Comment by rahul286** [comment]
Can you explain in more details then? 

By webmail, we meant inbound emails in general. 

If you want an interface to capture outgoing email, EE v4 already has support for mailhog.

**Comment by mbtamuli** [comment]
**tl;dr** - No inbound emails or email piping to any file(the kind that you mention). Outbound emails will work, after this PR is merged - https://github.com/EasyEngine/site-wp-command/pull/4

This might not be a webmail related issue, but we're not going to be setting up postfix or a similar tool to receive incoming mail. Support for that is going away in v4, at least from the core. 

If someone wants to support full mail server configuration, there can always be a community package for that.

**Comment by s-a-s-k-i-a** [comment]
Ohmy.. I wasn't aware. I really like th ease of setting up a full mailserver etc stuff with ee.

**Comment by rahul286** [comment]
> I had no idea you were dropping postfix/amavis/dovecot/etc

We're not dropping support for postfix. It's needed to ensure mail can be delivered from the EE sites as well as system-related emails.

Other than that we've dropped support for multiple cache types. We'll only support redis caching going forward in v4.

We are posting all updated on our blog http://easyengine.io/blog/ . You may signup for newsletter too https://rt.cx/subscribe
